### PR TITLE
command can't be separate by && in powershell

### PR DESCRIPTION
**Pull request by fgregg**

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 environment:
   password:
-    secure: uOcv5TOm/CgwOmiLQhNX+g==
+    secure: encrypted_pypi_password
```
Comment (fgregg) [inline appveyor.yml]: @kmike, I submitted a PR to crfsuite that will allow compiling crfsuite for python 2.6-3.5. In the meantime, we could release window wheels for all the python's except 3.5.

If you want to do that, you need to create an encrypted environmental variable https://ci.appveyor.com/tools/encrypt

like 

password=your_pypi_password

and put the whole string you get here:

Comment (kmike) [inline appveyor.yml]: great, thanks a lot @fgregg!

**Comment by fgregg** [comment]
Anything else I can help with?

**Comment by kmike** [comment]
Thanks for the ping :)

**Pull request merged**